build-scripts/swaync.rpkg.spec

```diff
@@ -1,34 +1,73 @@
 # vim: syntax=spec
+%global alt_pkg_name swaync
+
 Name:       {{{ git_dir_name }}}
 Version:    0.9.0
 Release:    1%{?dist}
 Summary:    Notification daemon with GTK GUI
 Provides:   desktop-notification-daemon
+Provides:   sway-notification-center = %{version}-%{release}
+Provides:   %{alt_pkg_name} = %{version}-%{release}
 License:    GPLv3
 URL:        https://github.com/ErikReider/SwayNotificationCenter
 VCS:        {{{ git_dir_vcs }}}
 Source:     {{{ git_dir_pack }}}
 
-BuildRequires:    meson >= 0.51.0
-BuildRequires:    vala
-
-BuildRequires: gtk3-devel >= 3.22
-BuildRequires: gtk-layer-shell-devel >= 0.1
-Requires: dbus
-BuildRequires: glib2-devel >= 2.50
-BuildRequires: gobject-introspection-devel >= 1.68
-BuildRequires: libgee-devel >= 0.20
-BuildRequires: json-glib-devel >= 1.0
-BuildRequires: libhandy-devel >= 1.4.0
-BuildRequires: systemd-devel
-BuildRequires: systemd
-BuildRequires: scdoc
-BuildRequires: pulseaudio-libs-devel
+BuildRequires:  meson >= 0.51.0
+BuildRequires:  vala
+BuildRequires:  scdoc
+BuildRequires:  pkgconfig(gtk+-3.0) >= 3.22
+BuildRequires:  pkgconfig(gtk-layer-shell-0) >= 0.1
+BuildRequires:  pkgconfig(json-glib-1.0) >= 1.0
+BuildRequires:  pkgconfig(libhandy-1) >= 1.4.0
+BuildRequires:  pkgconfig(glib-2.0) >= 2.50
+BuildRequires:  pkgconfig(gobject-introspection-1.0) >= 1.68
+BuildRequires:  pkgconfig(gee-0.8) >= 0.20
+BuildRequires:  pkgconfig(bash-completion)
+BuildRequires:  pkgconfig(fish)
+BuildRequires:  pkgconfig(libpulse)
+BuildRequires:  systemd-devel
+BuildRequires:  systemd
+
+Requires:       dbus
 %{?systemd_requires}
 
 %description
 A simple notification daemon with a GTK gui for notifications and the control center
 
+%package bash-completion
+BuildArch:      noarch
+Summary:        Bash completion files for %{name}
+Provides:       %{alt_pkg_name}-bash-completion = %{version}-%{release}
+
+Requires:       bash
```

Comment:
Bash, zsh and fish shouldn't be required

Reply:
This is requirement for sub package with completion functions. Main package do not have any shell as requirement.